.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
can you copy the .gitignore content from android-architecture-components? that will remove everything in .idea

note also that you'll need to copy the ones in the project subdirectories

Reply:
:+1: